diff --git a/node_modules/@owowagency/react-native-tuya/android/build.gradle b/node_modules/@owowagency/react-native-tuya/android/build.gradle
index e148e50..37cb78a 100644
--- a/node_modules/@owowagency/react-native-tuya/android/build.gradle
+++ b/node_modules/@owowagency/react-native-tuya/android/build.gradle
@@ -18,11 +18,11 @@ buildscript {
 
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 35
 
     defaultConfig {
-        minSdkVersion 23
-        targetSdkVersion 25
+        minSdkVersion 24
+        targetSdkVersion 35
         versionCode 1
         versionName "1.0"
 
@@ -32,12 +32,26 @@ android {
             abiFilters "armeabi-v7a","arm64-v8a"
         }
     }
+    // flavorDimensions "appstore"
+    //   productFlavors {
+    //     play { dimension "appstore"
+    //     matchingFallbacks = ['play'] }
+    //     amazon { dimension "appstore"
+    //     matchingFallbacks = ['amazon', 'play'] }
+    // }
 
+
+    flavorDimensions "appstore"
+    productFlavors {
+    googlePlay { dimension "appstore" }
+    amazon { dimension "appstore"; matchingFallbacks = ['play', 'amazon'] }
+}
     packagingOptions {
         pickFirst 'lib/*/libc++_shared.so'
     }
 
     buildTypes {
+      debug {}
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
@@ -48,6 +62,9 @@ android {
       sourceCompatibility JavaVersion.VERSION_1_8
       targetCompatibility JavaVersion.VERSION_1_8
     }
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
 }
 
 configurations.all {
@@ -67,6 +84,7 @@ dependencies {
     implementation 'io.reactivex.rxjava2:rxjava:2.1.7'
     implementation 'org.eclipse.paho:org.eclipse.paho.android.service:1.1.1'
     implementation 'androidx.localbroadcastmanager:localbroadcastmanager:1.0.0'
+    implementation "com.facebook.react:react-android:${rootProject.ext.reactNativeVersion}"
 }
 
 repositories {
@@ -79,18 +97,3 @@ repositories {
     maven { url 'https://developer.huawei.com/repo/' }
     maven { url 'https://jitpack.io' }
 }
-
-allprojects {
-  repositories {
-    exclusiveContent {
-      filter {
-        includeGroup "com.facebook.react"
-      }
-      forRepository {
-        maven {
-          url "$rootDir/../node_modules/react-native/android"
-        }
-      }
-    }
-  }
-}
