diff --git a/node_modules/@owowagency/react-native-tuya/android/build.gradle b/node_modules/@owowagency/react-native-tuya/android/build.gradle
index e148e50..337364b 100644
--- a/node_modules/@owowagency/react-native-tuya/android/build.gradle
+++ b/node_modules/@owowagency/react-native-tuya/android/build.gradle
@@ -18,6 +18,7 @@ buildscript {
 
 
 android {
+  namespace 'com.owowagency.reactnativetuya'
     compileSdkVersion 31
 
     defaultConfig {
