diff --git a/node_modules/@react-native-picker/picker/android/build.gradle b/node_modules/@react-native-picker/picker/android/build.gradle
index b700d8c..df7a990 100644
--- a/node_modules/@react-native-picker/picker/android/build.gradle
+++ b/node_modules/@react-native-picker/picker/android/build.gradle
@@ -21,6 +21,7 @@ def getExtOrIntegerDefault(name) {
 apply plugin: 'com.android.library'
 
 android {
+  namespace 'com.reactnativepicker'
   compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
   buildToolsVersion getExtOrDefault('buildToolsVersion')
 
@@ -29,6 +30,9 @@ android {
     targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
     vectorDrawables.useSupportLibrary=getExtOrDefault('vector_drawable_use_support_library')
   }
+      buildFeatures {
+        buildConfig = true
+    }
 }
 
 repositories {
