diff --git a/node_modules/expo-updates/android/build.gradle b/node_modules/expo-updates/android/build.gradle
index ed36c03..037a5ad 100644
--- a/node_modules/expo-updates/android/build.gradle
+++ b/node_modules/expo-updates/android/build.gradle
@@ -40,18 +40,16 @@ buildscript {
   }
 }
 
-// Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
-  from android.sourceSets.main.java.srcDirs
+    from android.sourceSets.main.java.srcDirs
+    archiveClassifier.set('sources')
 }
-
 afterEvaluate {
   publishing {
     publications {
       release(MavenPublication) {
-        from components.release
         // Add additional sourcesJar to artifacts
+        from components.findByName("release") ?: components.findByName("debug")
         artifact(androidSourcesJar)
       }
     }
@@ -64,6 +62,7 @@ afterEvaluate {
 }
 
 android {
+  namespace 'expo.modules.updates'
   compileSdkVersion safeExtGet("compileSdkVersion", 33)
 
   compileOptions {
@@ -71,6 +70,11 @@ android {
     targetCompatibility JavaVersion.VERSION_11
   }
 
+// ✅ Add this block to enable BuildConfig generation
+    buildFeatures {
+        buildConfig = true
+    }
+    
   kotlinOptions {
     jvmTarget = JavaVersion.VERSION_11.majorVersion
   }
