diff --git a/node_modules/expo-keep-awake/android/build.gradle b/node_modules/expo-keep-awake/android/build.gradle
index c7ed18d..dda49fc 100644
--- a/node_modules/expo-keep-awake/android/build.gradle
+++ b/node_modules/expo-keep-awake/android/build.gradle
@@ -35,10 +35,9 @@ buildscript {
   }
 }
 
-// Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
-  from android.sourceSets.main.java.srcDirs
+    from android.sourceSets.main.java.srcDirs
+    archiveClassifier.set('sources')
 }
 
 afterEvaluate {
@@ -59,6 +58,8 @@ afterEvaluate {
 }
 
 android {
+  namespace 'expo.modules.keepawake'
+
   compileSdkVersion safeExtGet("compileSdkVersion", 33)
 
   compileOptions {
@@ -79,6 +80,9 @@ android {
   lintOptions {
     abortOnError false
   }
+  publishing {
+        singleVariant("release") {}
+    }
 }
 
 dependencies {
