diff --git a/node_modules/lottie-react-native/android/build.gradle b/node_modules/lottie-react-native/android/build.gradle
index d3a398d..eb5799a 100644
--- a/node_modules/lottie-react-native/android/build.gradle
+++ b/node_modules/lottie-react-native/android/build.gradle
@@ -65,6 +65,9 @@ android {
     packagingOptions {
         exclude "**/libreact_render*.so"
     }
+    buildFeatures {
+        buildConfig = true
+    }
 
     packagingOptions {
         excludes = [
